apps/api/e2e/tests/boost-bonus.test.ts

```diff
@@ -0,0 +1,421 @@
+import { beforeAll, describe, test } from "vitest";
```

Comment:
wdyt about adding test case for duplicate wallet addresses in same batch. Should this be allowed or rejected?

What about max amount validation?